docs/portal.properties-Reference.md

```diff
@@ -66,6 +66,18 @@ app.name should be set to the name of the portal instance referenced in the "AUT
 The default view in OncoPrint ("patient" or "sample") can be set with the following option. The default is "patient".
 ```
 oncoprint.defaultview=sample
+```
+## Custom annotation of driver and passenger mutations
+cBioPortal allows for two different ways of displaying custom annotation of driver and passenger mutations in the MAF files (see [file formats](File-Formats.md#extending-the-maf-format). To enable the view of those annotations, you must specify a name for their labels (those labels appear in the oncoprint "Mutation color" menu):
+```
+oncoprint.custom_driver_annotation.binary.menu_label=Custom binary annotation
+oncoprint.custom_driver_annotation.class.menu_label=Custom tiers
+```
+
+By default, when at least one of the custom annotations is enabled, custom annotations are automatically selected, instead of OncoKB and Hotspots. If you do not wish to automatically select custom annotations, but select OncoKB and Hotspots instead, set this parameter to false:
```

Comment:
I would write: "If you wish to change this default behavior and have OncoKB and Hotspots as the default selected annotations, set this parameter to false:"